qpc/utils.py

```diff
@@ -151,11 +164,19 @@ def read_server_config():
         if use_http is None:
             use_http = True
 
+        if require_token is None:
+            require_token = True
+
         if not isinstance(use_http, bool):
             log.error('Server config %s has invalid value for use_http %s',
                       QPC_SERVER_CONFIG, use_http)
             return None
 
+        if not isinstance(require_token, bool):
+            log.error('Server config %s has invalid value for require_token %s',
+                      QPC_SERVER_CONFIG, require_token)
+            return None
```

Comment:
instead of returning `None` here could we just default it to True and still return the dictionary? that way we don't have to check for `None` in `read_require_auth`

Reply:
it returns true in that case.

Reply:
I get that, its just kind of a weird way around it. Like we could check for if the `require_token` is `None` or is not an instance of a bool & return `True` in both cases instead of `True` in one, and `None` in the other. I get that is how the other values were done but it seems odd.

Reply:
This is the pattern of that file.  We can open an issue to re-write it.  The others are similar.  Basically, if there is invalid JSON, it returns None for the entire config file.